tests/setup.js

```diff
@@ -37,11 +39,36 @@ Object.defineProperty(document, 'execCommand', {
 
 document.documentElement.style.fontSize = '12px';
 
+// isDependencyWarning returns true when the given console.warn message is coming from a dependency using deprecated
+// React lifecycle methods.
+function isDependencyWarning(params) {
```

Comment:
This isn't strictly necessary, but the previous version of this code still printed deprecation warnings from those components even if it prevented them from failing the tests. This will completely silence those errors while running tests so we only see useful output.